packages/updating-element/src/updating-element.ts

```diff
@@ -789,56 +589,38 @@ export abstract class UpdatingElement extends HTMLElement {
-   * a reactive property. In this case, pass no arguments. It should also be
-   * called when manually implementing a property setter. In this case, pass the
-   * property `name` and `oldValue` to ensure that any configured property
-   * options are honored.
-   *
-   * @param name {PropertyKey} (optional) name of requesting property
-   * @param oldValue {any} (optional) old value of requesting property
-   * @param options {PropertyDeclaration} (optional) property options to use
-   * instead of the previously configured options
-   */
+  /** @internal */
+  _propertyChanged(
+    name: PropertyKey,
+    oldValue: unknown,
+    options: PropertyDeclaration
+  ) {
+    super._propertyChanged(name, oldValue, options);
+    // Add to reflecting properties set.
+    // Note, it's important that every change has a chance to add the
+    // property to `_reflectingProperties`. This ensures setting
+    // attribute + property reflects correctly.
+    if (options.reflect === true && this._reflectingProperty !== name) {
+      if (this._reflectingProperties === undefined) {
+        this._reflectingProperties = new Map();
+      }
+      this._reflectingProperties.set(name, options);
+    }
+  }
+
   requestUpdate(
     name?: PropertyKey,
     oldValue?: unknown,
     options?: PropertyDeclaration
   ) {
-    let shouldRequestUpdate = true;
-    // If we have a property key, perform property update steps.
-    if (name !== undefined) {
-      options =
-        options ||
-        (this.constructor as typeof UpdatingElement).getPropertyOptions(name);
-      const hasChanged = options.hasChanged || notEqual;
-      if (hasChanged(this[name as keyof this], oldValue)) {
-        if (!this._changedProperties.has(name)) {
-          this._changedProperties.set(name, oldValue);
-        }
-        // Add to reflecting properties set.
-        // Note, it's important that every change has a chance to add the
-        // property to `_reflectingProperties`. This ensures setting
-        // attribute + property reflects correctly.
-        if (options.reflect === true && this._reflectingProperty !== name) {
-          if (this._reflectingProperties === undefined) {
-            this._reflectingProperties = new Map();
-          }
-          this._reflectingProperties.set(name, options);
-        }
-      } else {
-        // Abort the request if the property should not be considered changed.
-        shouldRequestUpdate = false;
-      }
-    }
-    if (!this.isUpdatePending && shouldRequestUpdate) {
+    const needsUpdate =
+      super.requestUpdate(name, oldValue, options) && !this.isUpdatePending;
+    if (needsUpdate) {
       this._updatePromise = this._enqueueUpdate();
     }
     // Note, since this no longer returns a promise, in dev mode we return a
     // thenable which warns if it's called.
-    return DEV_MODE ? requestUpdateThenable : undefined;
+    return DEV_MODE ? requestUpdateThenable : needsUpdate;
```

Comment:
Ugh, changing `requestUpdate` to return a boolean sort of messes up the DEV_MODE warning gambit; it's no longer safe to override `requestUpdate`, since in `DEV_MODE` you'll always get a sync-truthy thing.

Reply:
Hmm, good point. I can add `_scheduleUpdate` as an override point rather than using the return value.